### PR TITLE
Minor update in the Contributor's records to check if CI works

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Fixes #<NUM>

* Minor update in contributors.md and contributors record in the afterword

### List of changes proposed in this PR (pull-request)

* [ ] Minor update in the contributor's files

### What should a reviewer concentrate their feedback on?

- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: tests/no-bad-latin.py [hunk 1]
```diff
@@ -3,13 +3,9 @@
...
 
-HERE = os.getcwd()
-ABSOLUTE_HERE = os.path.dirname(HERE)
-IGNORE_LIST = [
-    os.path.join(ABSOLUTE_HERE, "book", "website", "_config.yml"),
-    os.path.join(ABSOLUTE_HERE, "book", "website", "community-handbook", "style.md"),
-    os.path.join(ABSOLUTE_HERE, "book", "website", "afterword", "contributors-record.md")
-]
+# HERE = os.getcwd()
+#ABSOLUTE_HERE = os.path.dirname(HERE)
```
Comment (malvikasharan) [inline tests/no-bad-latin.py:7]:
```suggestion
HERE = os.getcwd()
ABSOLUTE_HERE = os.path.dirname(HERE)
```

**Comment by malvikasharan** [comment]
@sgibson91 do you mind this change in the latin bot script? The only change is that the ignore list has just the file name, and when looking up for the ignored file, only the last string of the path is compared. It removes the error raised by full path issue.

I don't really understand why there were two consecutive 'the-turing-way' in the path error? can you explain me that?

**Comment by sgibson91** [comment]
@malvikasharan Fine by me! Yeah, I'm finding that Pull Request events in GitHub Actions are stranger and stranger the more I work with them 😫

**Review by sgibson91 — APPROVED** [review]
🎉

**Comment by malvikasharan** [comment]
Thanks Sarah. CIs are a whole new world for me.

**Pull request merged**